src/Clients/IThreadClient.cs

```diff
@@ -0,0 +1,10 @@
+using System.Threading.Tasks;
+using StreamChat.Models;
+
+namespace StreamChat.Clients
+{
+    public interface IThreadClient
```

Comment:
Please add a summary/remarks for the interface and the method. Example: https://github.com/GetStream/stream-chat-net/blob/master/src/Clients/IChannelClient.cs